gsa/src/gmp/models/cve.js

```diff
@@ -124,17 +124,13 @@ class Cve extends Info {
     if (isDefined(ret.raw_data) && isDefined(ret.raw_data.entry)) {
       const {entry} = ret.raw_data;
 
-      if (isDefined(ret.cwe)) {
-        ret.cweId = entry.cwe._id;
-      }
-
-      ret.publishedTime = parseDate(entry['published-datetime'].__text);
-      ret.lastModifiedTime = parseDate(entry['last-modified-datetime'].__text);
+      ret.publishedTime = parseDate(entry['published-datetime']);
+      ret.lastModifiedTime = parseDate(entry['last-modified-datetime']);
 
       ret.references = map(entry.references, ref => ({
         name: ref.reference.__text,
```

Comment:
This is still working with __text and ret.source is not?

Reply:
The problem is not that __text is not working anymore. The difference is having <foo>text</foo> and <foo id=123>text</foo> or <foo>text <bar /></foo>. For the first one __text does not work anymore but for the other two it does work and is actually needed.